build/common.sh

```diff
@@ -311,6 +311,7 @@ function kube::build::build_image() {
     api
     build
     cmd
+    contrib/mesos
```

Comment:
For the very short-term (so we can get this submitted now), please comment this out, and we can work on a good plan for making build/test of this code optional. I think there are some good reasons why we don't want to build/test this code by default:
1) it slows down the builds (pointed out by @thockin)
2) it pulls in a bunch of open-source dependencies; even though this code won't run unless the user is running k8s under mesos, I could see some security-conscious people being concerned; OTOH if this code only builds when specifically requested, it increases the security profile of k8s-by-itself slightly
3) although we should make every effort to avoid it, it is conceivable that we might temporarily need to break compatibility between the two codebases in the interest of expediency or exigency; making the default not try to build them together will limit the impact of these temporary events to only people using k8s-on-mesos, rather than everyone using k8s [of course this is a somewhat weak argument, as we could disable building them together in the same PR that introduces the incompatible change; but it might be confusing to users depending on how it is done]